lib/cn/kvs_mblk_desc.h

```diff
@@ -3,23 +3,70 @@
  * Copyright (C) 2015-2022 Micron Technology, Inc.  All rights reserved.
  */
 
-#ifndef HSE_KVS_MBLK_DESC_H
-#define HSE_KVS_MBLK_DESC_H
+#ifndef HSE_MBLK_DESC_H
```

Comment:
Why drop the KVS?

Reply:
Yeah, put it back plz!

Reply:
Granted, I hate the name "kvs_mblk_desc"

Reply:
An earlier version of this PR used `struct mblk_desc`, which is why function names and such are `mblk_`.  I backed off the struct rename because it resulted in 2X LOC change.  Happy to do that in a follow-on PR.